float32[6] actualTcpPose
float32[6] targetTcpPose
float32[6] actualTcpSpeed
float32[6] actualTcpForce
float32[6] actualQ
float32[6] actualQd
float32[6] targetQ
